nginx_config_reloader/__init__.py

```diff
@@ -313,7 +315,7 @@ def process_IN_DELETE(self, event):
             logger.warning("Configuration dir %s not found, waiting..." % dir_to_watch)
             time.sleep(5)
 
-        wm.add_watch(dir_to_watch, pyinotify.ALL_EVENTS, nginx_config_changed_handler)
+        wm.add_watch(dir_to_watch, pyinotify.ALL_EVENTS, nginx_config_changed_handler, rec=not no_recursive_watch)
```

Comment:
This looks like you did it the wrong way around :stuck_out_tongue: 
Can't you just make it `recursive_watch`? With default True.